wpilib/src/i2c.rs

```diff
@@ -151,11 +152,7 @@ impl I2C {
             .step_by(4)
             .zip((register_address..).step_by(4))
         {
-            let to_read = if expected.len() - i < 4 {
-                expected.len() - i
-            } else {
-                4
-            };
+            let to_read = cmp::min(4, expected.len() - i);
```

Comment:
Any particular reason here to not use Ord's min method directly?

Reply:
They're equivalent, I can switch it if it would be preferred